lib/src/main/java/uk/gov/di/ipv/core/library/domain/ErrorResponse.java

```diff
@@ -78,4 +80,19 @@ public String toJsonString() {
     public String getMessage() {
         return message;
     }
+
+    @JsonValue()
+    public int toValue() {
+        return this.getCode();
+    }
+
+    @JsonCreator
+    public static ErrorResponse forCode(int code) {
+        for (ErrorResponse element : values()) {
+            if (element.getCode() == code) {
+                return element;
+            }
+        }
+        return null;
```

Comment:
I think, we can use here Optional.empty() for protect to the NullPointerException(). It think, We can use the  Stream Api findFirst() for code reading